redi-connect-admin/src/App.js

```diff
@@ -220,163 +220,79 @@ const languages = [
+  {
+    id: 'mobileDevelopmentAndroid',
+    label: 'Android Mobile Development',
+    group: 'softwareEngineering',
+  },
+  { id: 'salesforce', label: 'Salesforce', group: 'softwareEngineering' },
+  { id: 'devOpsCloud', label: 'DevOps and Cloud (e.g. Azure, AWS)', group: 'softwareEngineering' },
+  { id: 'iot', label: 'IoT', group: 'softwareEngineering' },
+  { id: 'computerNetworking', label: 'Computer Networking', group: 'softwareEngineering' },
+  { id: 'blockchain', label: 'Blockchain', group: 'softwareEngineering' },
+  { id: 'productManagement', label: 'Product Management', group: 'otherProfessions' },
+  { id: 'projectManagement', label: 'Project Management', group: 'otherProfessions' },
+  { id: 'digitalMarketing', label: 'Digital Marketing', group: 'otherProfessions' },
+  { id: 'businessDevelopment', label: 'Business Development', group: 'otherProfessions' },
+  { id: 'sales', label: 'Sales', group: 'otherProfessions' },
+  { id: 'qualityAssurance', label: 'Quality Assurance', group: 'otherProfessions' },
+  { id: 'basicGerman', label: 'Basic German 🇩🇪', group: 'language' },
+  { id: 'businessGerman', label: 'Business German 🇩🇪', group: 'language' },
+  { id: 'english', label: 'English 🇬🇧', group: 'language' },
+  { id: 'graphicDesign', label: 'Graphic Design', group: 'design' },
+  { id: 'userInterfaceDesign', label: 'User Interface Design', group: 'design' },
+  { id: 'userExperienceDesign', label: 'User Experience Design', group: 'design' },
+  { id: 'motivationAndEncouragement', label: 'Motivation & encouragement', group: 'other' },
+  { id: 'friendAndHelp', label: 'Be a friend and help', group: 'other' },
+  { id: 'dontKnowYet', label: "I don't know yet", group: 'other' },
+  {
+    id: 'careerOrientationAndPlanning',
+    label: 'Career orientation & planning',
+    group: 'careerSupport',
+  },
+  {
+    id: 'internshipOrWorkingStudent',
+    label: 'Internship / working student position search',
+    group: 'careerSupport',
+  },
+  { id: 'jobSearch', label: 'Job search', group: 'careerSupport' },
+  {
+    id: 'jobApplicationsCvPreparationEnglish',
+    label: 'Job applications and CV preparation in English',
+    group: 'careerSupport',
+  },
+  {
+    id: 'jobApplicationsCvPreparationGerman',
+    label: 'Job applications and CV preparation in German',
+    group: 'careerSupport',
+  },
+  { id: 'interviewPreparation', label: 'Interview preparation', group: 'careerSupport' },
+  {
+    id: 'codingChallengePreparation',
+    label: 'Coding challenge preparation',
+    group: 'careerSupport',
+  },
+  {
+    id: 'buildingProfessionalNetwork',
+    label: 'Building a professional network',
+    group: 'careerSupport',
+  },
+  { id: 'entrepreneurship', label: 'Entrepreneurship', group: 'careerSupport' },
+  { id: 'freelancing', label: 'Freelancing', group: 'careerSupport' },
 ];
+const categoriesFlat = categories.map((cat) =>
+  Object.assign(cat, { label: `${cat.label} (${cat.group})` })
```

Comment:
Dont think this is a blocker more of a nit comment to use es6 syntax on this, would totally understand if you dont push this change ;)
```suggestion
  { ...cat, label: `${cat.label} (${cat.group})` }
```

Reply:
Heh, nice to have your pair of fresh eyes on this. I used `Object.assign()` here out of habit / old patterns. Completely makes sense. Thanks you Greek god <3